### PR TITLE
Update test.js to remove the test scenario where the API tries to translate without internet

### PR DESCRIPTION
**Pull request by coltongit**
~~Add a test scenario to attempt the translation of text without an internet connection as this was in the todo list codewise (it's meant to always fail, but that part of it has yet to be coded).~~
This has been removed as there's not much use to this test case anyway.

### CODE REVIEW

**Comment by vitalets** [comment]
Frankly speaking I don't see any profit from this test. I suggest just to remove it.

**Comment by coltongit** [comment]
Considering that it isn't easy to simulate the loss of an internet connection in JavaScript, it seems likely that this test case may have to be removed.

File: test.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ test.beforeEach(() => {
 
 test('translate without any options', async t => {
     const res = await translate('vertaler');
-
```
Comment (vitalets) [inline test.js]: Could you also keep original newlines. This is easier for reading tests.

**Comment by coltongit** [comment]
Not only did I add them back in, but I also added others to make it easier to read than it was beforehand.

File: test.js (hunk 1 above)
Comment (coltongit) [reply test.js]: This has already been done.

**Comment by coltongit** [comment]
Not only have the changes from this PR been moved to PR #15, but that PR also addresses an issue that had been previously resolved but was never merged. Due to this, this PR is hereby closed.

**Comment by coltongit** [comment]
Reopening as part of the separation requested in PR #15.

**Pull request merged**